recipes/python-omero/meta.yaml

```diff
@@ -19,14 +19,18 @@ requirements:
     - python
   run:
     - python
-    - zeroc-ice
+    - zeroc-ice >3.5,<3.7
 
 test:
   imports:
     - omero
     - omero.gateway
 
+extra:
```

Comment:
Instead of skipping linting it is better to update build.sh to include `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`